geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -297,6 +313,12 @@ pub trait GeyserPlugin: Any + Send + Sync + std::fmt::Debug {
         Ok(())
     }
 
+    /// Called when an entry is executed.
+    #[allow(unused_variables)]
+    fn notify_entry(&self, entry: ReplicaEntryInfoVersions) -> Result<()> {
```

Comment:
I think -- we should have an interface on check if entry is interested similar to transaction_notifications_enabled

Reply:
Yeah, agreed, that's a good idea.
How come account-notifications default to true and transaction-notifications default false? Should we have a check for block-notifications also?

Reply:
That is because we delivered accounts notification in the first release -- it is assumed on by default because that is the only thing being streamed. Transaction notification is added later and for backward compatibility we set ccount-notifications  to default as true.

Reply:
I see. And to my 2nd question?

Reply:
I considered block/slot as associated data when either accounts and transaction are transmitted hence did not have a separate knob for them.